event_listener.go

```diff
@@ -40,7 +41,8 @@ func NewEventListener(client EthClient,
 	return l
 }
 
-func (el *EventListener) Listen(fromBlock *big.Int, eventCh chan<- *ContractEvent, stop <-chan struct{}) error {
+func (el *EventListener) Listen(fromBlock *big.Int, stop <-chan struct{}) error {
```

Comment:
But Listen() will be blocked until listening service terminated.

Reply:
Since we have the whole instance of EventListener returned after New(), and since both logCh and eventCh are include in EventListener struct, caller can access eventCh directly.

Reply:
So how could caller access `eventCh`? It's a private variable.